diagnostics/diagnostics_metrics.go

```diff
@@ -0,0 +1,95 @@
+package diagnostics
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/stripe/veneur/v14"
+)
+
+var gitSha string = "git_sha:" + veneur.VERSION
+
+func CollectDiagnosticsMetrics(s *veneur.Server) {
```

Comment:
One last nit: could we pass in 
```
statsd *scopedstatsd.ScopedClient, interval time.Duration
```
to remove the dependency on "github.com/stripe/veneur/v14"?